components/service/glean/src/main/java/mozilla/components/service/glean/timing/TimingManager.kt

```diff
@@ -12,6 +12,11 @@ import mozilla.components.service.glean.private.CommonMetricData
 import mozilla.components.support.base.log.logger.Logger
 import java.util.WeakHashMap
 
+/**
+ * On opaque type that represents a glean timer identifier.
+ */
+typealias GleanTimerId = Long
```

Comment:
As @chutten pointed out correctly, this is not so much opaque, as this makes `Long` just as valid to pass in. An [inline class](https://kotlinlang.org/docs/reference/inline-classes.html) would be appropriate, but is experimental.